graph/patterns/dictionary.md

```diff
@@ -41,15 +41,23 @@ For more information, see the [OData reference](https://github.com/oasis-tcs/oda
 The following example demonstrates defining a dictionary that can contain string values.
 
 ```xml
-<ComplexType Name="stringDictionary" BaseType="graph.Dictionary">
-  <Annotation Term="Org.OData.Validation.V1.OpenPropertyTypeConstraint">
-    <Collection>
-      <String>Edm.String</String>
-    </Collection>
-  </Annotation>
-</ComplexType>
+<Schema Namespace="WorkloadNamespace">
```

Comment:
I think Dictionary needs to be defined in the core Microsoft.Graph namespace otherwise we will have multiple dictionary base types floating around.

Reply:
That was the intent, but I think you're right this isn't explicit enough. I've updated to declare a `microsoft.graph` namespace (and also confirms that the onboarding pipeline allows doing this...)